packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -136,6 +155,11 @@ export default class extends Component {
     }
   )
 
+  _setSchedulerGranularity = value =>
+    setSchedulerGranularity(this.props.host, value).then(() =>
+      this.setState({ schedGran: value })
```

Comment:
@MathieuRA @pdonias 

If the granularity change in the server side, the UI will not be updated. Do we want to let this behavior or add button/subscription to refresh this field? See https://github.com/vatesfr/xen-orchestra/pull/5320/files#r503812470

Reply:
If there's a simple way to have a subscription that doesn't flood the logs with errors in case we're on a host that doesn't support `getSchedulerGranularity`, then yes, it's probably better. Otherwise, it's fine like this.